WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -563,6 +569,9 @@ - (void)setEditing:(BOOL)editing animated:(BOOL)animated
     if (editing) {
         [self updateHeaderSize];
         self.tableView.tableHeaderView = self.headerView;
+
+        self.firstDisable = 0;
```

Comment:
> I don't know whether it's a good idea to store the date here as it consumes part of the timeout interval, from the moment "Edit" is pressed instead of the first disable.

Doh!  I missed that.  Yah, good point.  
And it looks like the updated code has `firstDisable` assigned a value of `nil` instead of `0`.  Perfect :)